src/main/java/sg/ncl/MainController.java

```diff
@@ -2220,12 +2224,27 @@ public String createExperiment(Model model, HttpSession session) throws WebServi
     @RequestMapping(value = "/experiments/create", method = RequestMethod.POST)
     public String validateExperiment(
             @ModelAttribute("experimentForm") ExperimentForm experimentForm,
-            HttpSession session,
             BindingResult bindingResult,
+            HttpSession session,
             final RedirectAttributes redirectAttributes) throws WebServiceRuntimeException {
 
         if (bindingResult.hasErrors()) {
             log.info("Create experiment - form has errors");
+            for (ObjectError objectError : bindingResult.getAllErrors()) {
+                FieldError fieldError = (FieldError) objectError;
+                switch (fieldError.getField()) {
+                    case "maxDuration":
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'MAX_DURATION' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)